src/features/embalm/stepContent/hooks/useCreateSarcohpagus.ts

```diff
@@ -53,6 +51,8 @@ export function useCreateSarcophagus() {
         // TODO: Use requiredArchaeologists value
         threshold: 3,
       });
+      //save shards to arch profiles
+      selectedArchaeologists.forEach((arch, index) => (arch.profile.shard = shards[index]));
```

Comment:
First, what is the purpose of adding the shards to the arch profile here? Does it have something to do with sending the double hashed shard in the `createSarcophagus` function? 

Also, this mutates react state which will cause unexpected bugs. To add shards to the arch profiles you will need add a custom dispatch action to set the state of the selected archaeologists, or something along those lines.

Reply:
Yeah, we need to save the shards so we can double hash them. I will write a set state for them. I will spend some time thinking about what is the correct data to save to the store here.